apis/discussionForum/discussionForum.yaml

```diff
@@ -0,0 +1,3952 @@
+openapi: 3.0.1
+info:
+  title: Discussions Middleware
+  description: The discussion forum microservice allows you to perform the nodebb
```

Comment:
please add why this is needed? what does it do?

Reply:
Could you also maintain the same name and title? Since the microservice is called Discussion Forum Service, Can the title of the API be 'Discussion Forum' and not 'Discussion Middleware'?